act-rules-format/act-rules-format.bs

```diff
@@ -452,15 +454,15 @@ Definitions can be put in the rule [glossary](#glossary), or they can be defined
 </aside>
 
 <aside class=example>
-  <header><strong>Incorrect Example:</strong>This example demonstrates an applicability that attempts to be subjective, but is so broad that it does not meet the "must be unambiguous" requirement. For example, would a text emoji like ":)" be considered non-text content that requires an accessible name? Due to the ambiguity in the applicability it is impossible to tell.</header>
+  <header><strong>Incorrect Example:</strong>This example demonstrates an applicability that attempts to be subjective, but does not meet the "must be unambiguous" requirement. For example, non-text content could refer to images, interactive elements, or even a text emoji like ":)". Due to the ambiguity in the applicability it is impossible to tell.</header>
```

Comment:
Do we need to add markup to that smiley?

Reply:
Add accessible name "Text smiley face created using a colon and parenthesis".